### ISSUE TITLE
Hard coded path calling CKEditor

### ISSUE THREAD

**Issue by emasters** [CCALI/caw#36]
Somewhere this is getting called: https://d7staging.calidev.org/caw/js/ckeditor_config.js?t=H4PG and failing with a 404. That path needs to set to a global that is defined in config.php or use one of the existing globals like `SITE_URL`

**Comment by emasters** [comment]
The problem is at https://github.com/CCALI/caw/blob/quiz-wright/js/scripts.js#L52. 
I changed the path from `/caw` to `/quizwright` to get it to work but it needs to abstracted out to a config file.

**Comment by sgoshorn** [comment]
Good idea. For reference: CKEditor needs a full path like  `/caw/js/ckeditor_config.js` instead of relative like `./js/ckeditor_config.js` because I'm using the CDN version. A relative path tells CK to look for the config on the CDN which won't work for our customization. Alternative is to make all the CK files local but that is just more files to manage.
